data/loftwah.json

```diff
@@ -1,31 +1,30 @@
 {
   "name": "Dean Lofts",
   "type": "personal",
+  "displayStatsPublic": true,
   "bio": "DevOps Engineer, Music Producer, and big fan of Open Source.",
   "avatar": "https://github.com/loftwah.png",
+  "tags": [
+    "AWS",
+    "Beats",
+    "DevOps",
+    "Music",
+    "Open Source",
+    "Ruby",
```

Comment:
The trailing comma is not needed

Reply:
Cheers, Eddie! I wish JSON would have made things work this way 😅 I'll fix it up today.